embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/arch/schema/SchemaType.kt

```diff
@@ -195,6 +197,20 @@ internal sealed class SchemaType(
         ).toNonNullMap()
     }
 
+    internal class NetworkRequest(networkRequest: EmbraceNetworkRequest) : SchemaType(EmbType.Performance.Network) {
+        override val attrs = mapOf(
+            "url.full" to networkRequest.url,
+            "http.request.method" to networkRequest.httpMethod,
+            "http.response.status_code" to networkRequest.responseCode,
+            "http.request.body.size" to networkRequest.bytesSent,
+            "http.response.body.size" to networkRequest.bytesReceived,
+            "error.type" to networkRequest.errorType,
+            "error.message" to networkRequest.errorMessage,
+            "emb.w3c_traceparent" to networkRequest.w3cTraceparent,
+            "emb.trace_id" to getValidTraceId(networkRequest.traceId),
```

Comment:
we should rename this as it is confusing in terms of what traceId means in OTel.